package.json

```diff
@@ -525,6 +534,12 @@
         "command": "workbench.view.extension.objectExplorer",
         "key": "ctrl+alt+d",
         "mac": "cmd+alt+d"
+      },
+      {
+        "command": "mssql.copyObjectName",
+        "key": "f2",
+        "mac": "f2",
```

Comment:
This is how you set keybindings - https://code.visualstudio.com/api/references/contribution-points#contributes.keybindings

Reply:
The platform-specific bindings are optional. 

https://github.com/microsoft/azuredatastudio/blob/master/src/vs/workbench/services/keybinding/browser/keybindingService.ts#L56

If not specified it'll use the primary one (key). So duplicating it here is unnecessary.

Reply:
It's good to know the internal working of the service, but the official API shows to use both fields, and as an extension user, I'll use the API as it's suggested.

Reply:
Fine, it's not going to break anything so you can leave it. But I would assume that the example is showing both because they're different keybindings (`ctrl+f1` vs `cmd+f1`) and not because they're suggesting to put it there all the time.

(Linux and windows specific keybindings are also supported yet the example doesn't show those)